tests/integration/test_router_keyword_inference.py

```
@@ -0,0 +1,82 @@
+
+
+def _resolve_powershell() -> str | None:
+    return shutil.which("pwsh") or shutil.which("powershell")
+
+
+def _invoke_route(prompt: str) -> dict[str, object]:
+    powershell = _resolve_powershell()
+    if not powershell:
+        pytest.skip("PowerShell executable not available in PATH")
+
+    completed = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)